src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql

```diff
@@ -183,6 +185,10 @@ parameter InclusionMedicationsLookbackPeriod default 180 days //DO WE NEED THIS?
 
 //system url from PDMP identifier
 define PDMP_SYSTEM_VALUE: 'https://github.com/uwcirg/script-fhir-facade/'
+//system url for labcorp
+define LAB_CORP_SYSTEM_VALUE: 'https://www.labcorp.com/tests'
+//lab panel test order code from lab corp
+define Urine_Screen_Panel_Test_Order_Code: '733727'
```

Comment:
This is one of TWO panels they use.  Please add `763824`

Reply:
fixed.